tests/unit/backend/preparation/test_normalization.py

```diff
@@ -124,6 +124,24 @@ def test_normalization_bright_dark():
     assert diff < 0.01
 
 
+def test_normalization_no_darks():
```

Comment:
Do you know about [numpy.testing](https://numpy.org/doc/stable/reference/routines.testing.html)?

Reply:
I am assuming you are asking to use np.testing.assert_all_close instead of computing the average difference here. They are similar in function, but I prefer the explicit one here over the one provided by numpy